package.json

```diff
@@ -66,7 +66,7 @@
     "jquery-ui-bundle": "^1.12.1-migrate",
     "reconnecting-eventsource": "^1.1.0",
     "ssri": "^8.0.1",
-    "svelte-materialify": "^0.3.6",
+    "svelte-materialify": "^0.3.11",
```

Comment:
Keep it at the old materialify, the latest materialify breaks something in our project irc. We'll be switching to something more stable like smelte right after this too.